src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowPredicate.java

```diff
@@ -621,6 +622,10 @@ public static final boolean getPredicateTakenOfflineByStaffAfterNotificationAckn
             && caseData.getApplicant1ResponseDeadline().isBefore(LocalDateTime.now())
             && caseData.getApplicant1ProceedWithClaim() == null;
 
+    public static final Predicate<CaseData> demageMultiClaim = caseData ->
+        AllocatedTrack.MULTI_CLAIM.equals(caseData.getAllocatedTrack())
+        && CaseCategory.UNSPEC_CLAIM.equals(caseData.getCaseAccessCategory());
```

Comment:
Does this not also impact SPEC Claims too? 
Claim Type should not matter - all Multi Track should drop offline after issue

Reply:
@HarryH96 at this ticket is for Demages thats why I am setting MULTI_CLAIM_YES only if claim is demages and Multiclaim.